packages/cli/src/License.ts

```diff
@@ -79,6 +84,15 @@ export class License {
 		}
 	}
 
+	async collectUsageMetrics() {
+		return [
+			{
+				name: 'activeWorkflows',
+				value: await Container.get(WorkflowRepository).count({ where: { active: true } }),
+			},
```

Comment:
> Products can be configured to support the collection of usage metrics from consumers. `product.metrics` is an array of `{name: string, type: 'counter' | 'gauge'}` objects

Why the difference between `value` and `type`? Are the docs outdated?

Reply:
`product.metrics` refers to the metric **definitions** maintained in License Server, whereas the SDK's `collectUsageMetrics()` callback expects an array of name / value pairs.